EIPS/eip-7668.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 7668
+title: Add Max Epoch Churn Limit
+description: Modify the churn limit function to upper bound it to a max value
+author: dapplion (@dapplion)
+discussions-to: https://ethereum-magicians.org/t/eip-7668-add-max-epoch-churn-limit/15709
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-09-07
+---
+
+## Abstract
+
+Limit Beacon chain growth from exponential to linear to mitigate the risks of very high levels of total ETH supply staked.
+
+## Motivation
+
+With a full deposit queue, the share of ETH supply staked will reach 50% by May 2024, 75% by September 2024, and 100% by December 2024. While yields decrease as the validator set size increases, the APY at 100% ETH supply staked is 1.6% for consensus rewards only excluding MEV. This yield can be sufficient to attract capital such that the equilibrium point of the validator set size is close to its maximum possible.
+
+Reaching very high levels of total ETH supply staked can have negative externalities. Liquid staking derivatives (LSDs) are likely to absorb most of the staking demand, jeopardizing the economic guarantees of the Beacon chain. If LSD tokens overtake the native token in liquidity, they permeate systemic risks into most Ethereum applications.
```

Comment:
In this EIP, I might stay closer to the system impact of the linear load (state size and attestations) and the security and economic impacts more broadly of 50%+ staked. I don't think there are 100% clear answers but there is a lot of worry and debate at these thresholds, so instead of making explicit claims about a particular thing -- e.g., LSDs -- I think it simpler to make claims more broadly about the unintended consequences of very high stake rates.

Also, on this particular item, it's not clear that just reducing total stake rate would reduce the % that goes into LSDs and I don't think you should attempt to make that argument here

```suggestion
Reaching very high levels of total ETH supply staked can have negative externalities. Liquid staking derivatives (LSDs) might continue to grow at outsized pace, jeopardizing the economic guarantees of the Beacon chain. If LSD tokens overtake the native token in liquidity, they permeate systemic risks into most Ethereum applications.
```

Reply:
Removed LSD claims from EIP text